tests/unit/detection/test_detection_notebooks.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This test is based on the test suite implemented for Recommenders project
+# https://github.com/Microsoft/Recommenders/tree/master/tests
+
+import os
+import papermill as pm
+import pytest
+import scrapbook as sb
+
+# Unless manually modified, python3 should be
+# the name of the current jupyter kernel
+# that runs on the activated conda environment
+KERNEL_NAME = "python3"
+OUTPUT_NOTEBOOK = "output.ipynb"
+
+
+@pytest.mark.notebooks
+def test_00_notebook_run(detection_notebooks):
+    notebook_path = detection_notebooks["00"]
+    pm.execute_notebook(
+        notebook_path,
+        OUTPUT_NOTEBOOK,
+        parameters=dict(PM_VERSION=pm.__version__),
+        kernel_name=KERNEL_NAME,
+    )
+
+    nb_output = sb.read_notebook(OUTPUT_NOTEBOOK)
+    assert len(nb_output.scraps["detection_bounding_box"].data) > 0
+
+
+@pytest.mark.notebooks
+def test_01_notebook_run(detection_notebooks, tiny_od_data_path):
+    notebook_path = detection_notebooks["01"]
+    pm.execute_notebook(
+        notebook_path,
+        OUTPUT_NOTEBOOK,
+        parameters=dict(
+            PM_VERSION=pm.__version__, DATA_PATH=tiny_od_data_path, EPOCHS=1
```

Comment:
Similar to above, this might be too slow for CPU-based unit tests. If so, could reduce image resolution to 100 pixels, reduce number of proposals, and only use a couple of training images.